cmd/libsnap-confine-private/cgroup-support.c

```diff
@@ -32,6 +34,9 @@
 #include "string-utils.h"
 #include "utils.h"
 
+static const char *cgroup_dir = "/sys/fs/cgroup";
+static const char *snapd_cgroup_dir = "/sys/fs/cgroup/snapd";
```

Comment:
Shouldn't we use a path under `/run/snapd` so that we don't cause an awkward situation on a v2 system, where there's a /sys/fs/cgroup v2 mount point, and a /sys/fs/cgroup/snapd named v1 hierarchy inside?

Reply:
Hmmm, that's an interesting point. Perhaps we should. I will think about this some more (mainly about consequences of cgroup outside of /sys/fs/cgroup)

Reply:
I thought about this some more and realized we cannot use `/sys/fs/cgroup/snapd`. In all the experiments we used custom locations like `/mnt/foo` so we didn't bump into this. I changed the path to `/run/snapd/cgroup`.